fix: Doctor Core - Fix for intermittent exception while downloading VDRs.(#ENGCE-12018)

HarikrishnaV opened the pull request:
## Non-Customer Highlights
* Fix for the following exception while downloading VDRs.
`TypeError: Cannot read property 'message' of undefined while exporting VDRs`

HarikrishnaV committed:
fix: Doctor CLI - Fix for Intermittent exception TypeError: Cannot read property 'message' of undefined while downloading VDRs.(#ENGCE-12018)

mukundbayyaram commented:
@HarikrishnaV Were you able to recreate the error in local? Is it because error message was different intermittently from soba?
Also, can you bump version in `package.json`?

HarikrishnaV committed:
fix: bumped version in package.json

HarikrishnaV commented:
> @HarikrishnaV Were you able to recreate the error in local? Is it because error message was different intermittently from soba? Also, can you bump version in `package.json`?

@mukundbayyaram I am unable to reproduce the same in local but as per the error log, I see typeError for `message` reading from undefined. So, I have added condition over there.  
For VDR's export the error message is `VDR {vdrName} doesn't exist`. I have updated one more regex condition to check this as well.

HarikrishnaV committed:
fix: Removed un-necessary check.
